test/Validate.Capability.cpp

```diff
@@ -194,6 +194,68 @@ const vector<string>& AllCapabilities() {
   return *r;
 }
 
+const vector<string>& AllV10Capabilities() {
```

Comment:
It would be, particularly if `AllCaps` could still rely on static initialization in that case.  Otherwise, there'll be overhead whenever it's used, and it's used in nearly every test...

We have a similar pattern in `MatrixDependencies` vs. `ShaderDependencies`.